index.html

```diff
@@ -1,237 +1,258 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Raf's Portfolio</title>
+  <title id="Portfolio">Raf's Portfolio</title>
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job working on this milestone! 💯 However, ids won't make your title more accessible, as they are element identifiers that you can use as targets for CSS selectors, anchor tags, or javascript. At the moment you have a good title that describes briefly the content of your page, so you can safely remove this `id`. 🤓 🛠️